tests/run_tests.py

```diff
@@ -209,6 +217,7 @@ def test():
 
     if args.cpp:
         _test_cpp()
+        return
```

Comment:
With `--cpp=True` as the default value, this might turn off all the python tests:

<img width="883" alt="image" src="https://user-images.githubusercontent.com/22334008/170613762-4e705315-4997-4ac1-8f63-306ee374197d.png">